nix/pkgs.nix

```diff
@@ -0,0 +1,52 @@
+{
+  sources ? import ./sources.nix,
+  config ? {},
+  system ? builtins.currentSystem,
+  overlays ? []
+}:
+let
+  allOverlays =
+    # These overlays augment centrally defined packages with things specific
+    # to this service.
+    [
+      (self: super: {
+        ci = {
+          pr-step = self.callPackage
+            ({
+              lib,
+              utils,
+              yarn-berry,
+              nodejs,
+              name ? "pr-step"
+            }:
+            let
+              yarnExe = lib.getExe yarn-berry;
+            in utils.writeBashBin
+              name
+              ''
+              set -ueo pipefail
+
+              echo yarn version: $(${yarnExe} --version)
+
+              ${yarnExe} --immutable
```

Comment:
is this expected to run in nix sandbox? since its not using nixify it'll fail without

Reply:
No sandbox